core/include/drivers/regulator.h

```diff
@@ -286,10 +284,7 @@ static inline TEE_Result regulator_set_min_voltage(struct regulator *regulator)
  * regulator_get_voltage() - Get regulator current level in microvolt
```

Comment:
I think we should rephrase this to `Get regulator effective voltage level in microvolt`.
but that out of the scope of your P-R.

Reply:
done